components/cam/src/physics/crm/cam_diagnostics.F90

```diff
@@ -0,0 +1,2120 @@
+module cam_diagnostics
```

Comment:
So my understanding is that /cam/cam_diagnostics.F90 is directly from the E3SM repo, while  /crm/cam_diagnostics.F90 is from the ECP repo. And /crm/cam_diagnostics.F90 overrides /cam/cam_diagnostics.F90 when SP_CAM=TRUE. Therefore, /crm/cam_diagnostics.F90 should be basically /cam/cam_diagnostics.F90  + anything related with MMF added on.  I have 2 questions related with this. 
/cam/cam_diagnostics.F90 is updated with 2 new fields, U90M and V90M. Don't they need to be added in /crm/cam_diagnostics.F90  as well? Another question is, how will /crm/cam_diagnostics.F90 be managed/updated when there's a change in /cam/cam_diagnostics.F90?

Reply:
We should probably let the original cam_diagnostics.F90 be active and then just write a new module called "mmf_diagnostics" or something like that to add the extra MMF related fields.

Reply:
that sounds like a better approach, at least it will be easier to maintain the crm side code. The same change needs to take place for clouds_diagnostics.F90

Reply:
I see many files that exist in both /cam/src/physics/cam and /cam/src/physics/crm. Those files residing under physics/cam are from E3SM repo and those inside physics/crm are from ECP repo. Do we have any strategy to update files under physics/crm whenever there's any changes made in the files in physics/cam?

Reply:
I can make those new modules.
The strategy is to manually maintain those files when there are differences. As we work through how to remove the offline physics calls we'll definitely reduce the amount of redudant code, but that will likley take us awhile.

Reply:
ok. sounds good.

Reply:
My larger vision is to let cam_diagnostics mostly die, same for clouds, and only output things specific to the CRM physics package. This would avoid the problem we currently have of trying to understand, for example, whether CLDLOW represent low cloud diagnosed from the CRM cloud fraction, or from the CAM cloud fraction. I think a lot of variables in clouds_diagnostics.F90 should be written by crm_physics.F90, and likewise for cam_diagnostics.F90.

Reply:
So the only MMF specific stuff in this file is just for MAML, which includes some ifdef/else logic, so we should just leave this for now and deal with it when we fix the MAML functionality on the E3SM side.

Reply:
ok